pkg/vm/engine/disttae/txn.go

```diff
@@ -435,6 +435,10 @@ func needRead(expr *plan.Expr, blkInfo BlockMeta, tableDef *plan.TableDef, proc
 	if expr == nil {
 		return true
 	}
+	// return true anyway
```

Comment:
What are we doing here?   Regardless of expr is nil or not we return true?   What follows are all dead code?

Reply:
@nnsgmsone asked for this modification, needRead always returns true for now, otherwise, there will be panic

Reply:
OK.   I guess we need his approval on this.

Reply:
> OK. I guess we need his approval on this.

Sadly, I just found the cause of the blockread panic yesterday. That zonemap I haven't had time to debug to locate the problem. Now the zonemap is not working, there should be some bugs. @fengttt